iFixit/react-commerce
# Add product section

This PR adds the first section of the product page with the following features:

- Product gallery (that is in sync with the product options selection)
- Title and pricing
- Product options
- Add to cart (the button doesn't automatically open the cart drawer, this will be done in a separate PR)
- Inventory details
- Product description 
- Kit content
- Product spec
- Warranty
- Prop 65 warnings

The section is not responsive yet, and it's not necessarily super polished. The aim of this PR is to merge the bulk of it, and gradually refine it in follow up PR.

## QA

1. Visit Vercel Preview
2. Go to [/Product/iphone-11-pro-screen](https://react-commerce-git-add-product-section-ifixit.vercel.app/Products/iphone-11-pro-screen)
3. Verify that the features described above are present
